### PR TITLE
[mod_search]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_captcha_recaptcha
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_search/mod_search.xml [hunk 1]
```diff
@@ -26,122 +26,139 @@
...
 					label="COM_MODULES_FIELD_CACHE_TIME_LABEL"
-					description="COM_MODULES_FIELD_CACHE_TIME_DESC" />
+					description="COM_MODULES_FIELD_CACHE_TIME_DESC"
+					default="900"
+				/>
+
 				<field
 					name="cachemode"
 					type="hidden"
 					default="itemid">
```
Comment (andrepereiradasilva) [inline modules/mod_search/mod_search.xml]: shouldn't the > be in a new line?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 37cb2adf968b6df74a2c01d9e1440379b02d8684<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10591">issues.joomla.org/joomla-cms/10591</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on 37cb2adf968b6df74a2c01d9e1440379b02d8684<br /><br />on review - Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10591">issues.joomla.org/joomla-cms/10591</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10591">issues.joomla.org/joomla-cms/10591</a>.</sub>

**Pull request closed without merging**